Base/usr/share/man/man7/ramfs.md

```diff
@@ -0,0 +1,18 @@
+## Name
+
+ramfs - SerenityOS RAMFS
+
+## Description
+
+The kernel can mount a purely RAM-backed filesystem.
+This filesystem is normally mounted in /tmp and /dev.
+
+## Filesystem-Specific Mount flags
+
+- `fs_max_size` - Given an unsigned integer with this flag, the new RAMFS instance will be
```

Comment:
Do we want to just name this `max_size`? The `fs_` part is kind of implied as far as I can tell.